src/CoreLocation/CLLocationManager.cs

```diff
@@ -47,5 +47,22 @@ public static bool IsMonitoringAvailable (Type t)
 			return false;
 		}
 #endif
+
+		public static CLAuthorizationStatus Status {
+			get {
+				// because an instance property in iOS 14, tvOS 14, Watch 7 and macOS 10,16
+#if IOS || TVOS
+				if (UIKit.UIDevice.CurrentDevice.CheckSystemVersion (14, 0)) {
+#elif WATCH
+				if (WatchKit.WKInterfaceDevice.CurrentDevice.CheckSystemVersion (7, 0)) {
+#else 
+				if (PlatformHelper.CheckSystemVersion (10, 16)) {
```

Comment:
This would make things easier if we get new platforms we need to support, since we'll get compilation errors instead of having to wait for someone to run into the potentially incorrect version check and file an issue about it:

```suggestion
#elif MONOMAC
				if (PlatformHelper.CheckSystemVersion (10, 16)) {
#else 
				#error Unknown platform
```